firefox-ios/Client/Frontend/Browser/BrowserViewController/Views/BrowserViewController.swift

```diff
@@ -553,7 +553,10 @@ class BrowserViewController: UIViewController,
             }
 
             // Update states for felt privacy
-            updateInContentHomePanel(tabManager.selectedTab?.url)
+            if state.webviewURL != nil {
+                updateInContentHomePanel(state.webviewURL)
+            }
```

Comment:
Should we update even if the URL is nil? If we have a blank tab, this update would be necessary (i.e. a blank tab shouldn't have a reload button as there is no URL, and if we have a blank tab we should ensure we are see the webview as "content homepanel")

Reply:
I always forget that blank tab exists 🙈 

This change scares me a little and it adds bugs to private mode so I'm going to suggest that we just go with your PR for now for 123.2. I'm just worried there are other repercussions for spamming this call and even other cases where the webview being nil could cause issues.

Reply:
I was also unaware of the blank tab! How do I trigger that, do you have a screenshot of what that looks like?